templates/elasticsearch.yml.j2

```diff
@@ -35,6 +35,10 @@ action.auto_create_index: false
 action.auto_create_index: {{ es_action_auto_create_index }}
 {% endif %}
 
+{% if es_processors is defined %}
```

Comment:
What is the reason for not just specifying this in the `es_config` variable? Talking about doing it like this:

```
    es_config:
      processors: 16
```